trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.length > 10 ? @nameable.correct_name[0, 9] : @nameable.correct_name
+  end
+end
```

Comment:
- [ ] Based on the requirements, the `corret_name` method of the `TrimmerDecorator` class should return the first **_ten_** characters of the nameable's name. Yours currently returns the first _**nine**_ characters. Kindly implement this change. 👍 😎